src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
```

Comment:
Ensure that variables are defined before being used. If `$border-radius` is not defined in `src/styles/utils/variables.scss`, it will cause an error. Verify that it is indeed declared and has an appropriate value.